epoxy-processortest/src/test/resources/ModelReturningClassType.java

```diff
@@ -0,0 +1,14 @@
+package com.airbnb.epoxy;
+
+public class ModelReturningClassType extends EpoxyModel<Object> {
+  @EpoxyAttribute int value;
```

Comment:
maybe add another field and setter for it to test the `removeMethodIfDuplicatedBySetter` code?